test/constraint.jl

```diff
@@ -266,31 +283,36 @@ function test_indicator_constraint(ModelType, ::Any)
     @variable(model, y)
     for cref in [
         @constraint(model, a => {x + 2y <= 1}),
-        @constraint(model, a ⇒  {x + 2y ≤ 1})
+        @constraint(model, a ⇒ {x + 2y ≤ 1})
     ]
         c = JuMP.constraint_object(cref)
         @test c.func == [a, x + 2y]
         @test c.set == MOI.IndicatorSet{MOI.ACTIVATE_ON_ONE}(MOI.LessThan(1.0))
     end
     for cref in [
-        @constraint(model, !b => {2x + y <= 1});
-        @constraint(model, ¬b ⇒  {2x + y ≤ 1});
+        @constraint(model, !b => {2x + y <= 1})
+        @constraint(model, ¬b ⇒ {2x + y ≤ 1})
         # This returns a vector of constraints that is concatenated.
-        @constraint(model, ![b, b] .=> {[2x + y, 2x + y] .≤ 1});
+        @constraint(model, ![b, b] .=> {[2x + y, 2x + y] .≤ 1})
     ]
         c = JuMP.constraint_object(cref)
         @test c.func == [b, 2x + y]
         @test c.set == MOI.IndicatorSet{MOI.ACTIVATE_ON_ZERO}(MOI.LessThan(1.0))
     end
-    err = ErrorException("In `@constraint(model, !(a, b) => {x <= 1})`: Invalid binary variable expression `!(a, b)` for indicator constraint.")
+    err =
+        ErrorException("In `@constraint(model, !(a, b) => {x <= 1})`: Invalid binary variable expression `!(a, b)` for indicator constraint.")
```

Comment:
It seems like if you drop down a line, but that line exceeds the line length due to a string, you shouldn't drop down the line.

Reply:
Potentially: https://github.com/domluna/JuliaFormatter.jl/issues/336